Slider/tests/Slider-specs.js

```diff
@@ -1,512 +1,441 @@
+	});
+
+	test('should decrement the value of horizontal slider on key left when active', () => {
+		render(
+			<Slider activateOnSelect defaultValue={50} />
+		);
+		const slider = screen.getByRole('slider');
+
+		activate(slider);
+		leftKeyDown(slider);
+
+		const expectedAttribute = 'aria-valuetext';
+		const expectedValue = '49';
+
+		expect(slider).toHaveAttribute(expectedAttribute, expectedValue);
+	});
+
+	test('should decrement the value of horizontal slider on key left', () => {
+		render(
+			<Slider defaultValue={50} />
+		);
+		const slider = screen.getByRole('slider');
+
+		focus(slider);
+		leftKeyDown(slider);
+
+		const expectedAttribute = 'aria-valuetext';
+		const expectedValue = '49';
+
+		expect(slider).toHaveAttribute(expectedAttribute, expectedValue);
+	});
+
+	test('should decrement the value of vertical slider on key down when active', () => {
+		render(
+			<Slider activateOnSelect defaultValue={50} orientation="vertical" />
+		);
+		const slider = screen.getByRole('slider');
+
+		activate(slider);
+		downKeyDown(slider);
+
+		const expectedAttribute = 'aria-valuetext';
+		const expectedValue = '49';
+
+		expect(slider).toHaveAttribute(expectedAttribute, expectedValue);
+	});
+
+	test('should decrement the value of vertical slider on key down', () => {
+		render(
+			<Slider defaultValue={50} orientation="vertical" />
+		);
+		const slider = screen.getByRole('slider');
+
+		focus(slider);
+		downKeyDown(slider);
+
+		const expectedAttribute = 'aria-valuetext';
+		const expectedValue = '49';
 
-			activate(subject);
-			leftKeyDown(subject);
+		expect(slider).toHaveAttribute(expectedAttribute, expectedValue);
+	});
+
+	test('should increment the value of horizontal slider on key right when active', () => {
+		render(
+			<Slider activateOnSelect defaultValue={50} />
+		);
+		const slider = screen.getByRole('slider');
+
+		activate(slider);
+		rightKeyDown(slider);
+
+		const expectedAttribute = 'aria-valuetext';
+		const expectedValue = '51';
+
+		expect(slider).toHaveAttribute(expectedAttribute, expectedValue);
+	});
+
+	test('should increment the value of horizontal slider on key right', () => {
+		render(
+			<Slider defaultValue={50} />
+		);
+		const slider = screen.getByRole('slider');
+
+		focus(slider);
+		rightKeyDown(slider);
+
+		const expectedAttribute = 'aria-valuetext';
+		const expectedValue = '51';
+
+		expect(slider).toHaveAttribute(expectedAttribute, expectedValue);
+	});
+
+	test('should increment the value of vertical slider on key up when active', () => {
+		render(
+			<Slider activateOnSelect defaultValue={50} orientation="vertical" />
+		);
+		const slider = screen.getByRole('slider');
+
+		activate(slider);
+		upKeyDown(slider);
+
+		const expectedAttribute = 'aria-valuetext';
+		const expectedValue = '51';
+
+		expect(slider).toHaveAttribute(expectedAttribute, expectedValue);
+	});
+
+	test('should increment the value of vertical slider on key up when', () => {
```

Comment:
the last word "when" seems to be extra. It's an old bug, but we can fix it now :)

Reply:
Done